src/subcommandlineparser.cpp

```diff
@@ -61,7 +61,7 @@ static const std::map<QString, QList<QCommandLineOption>> options{
                       "The path does not need to be unique. "
                       "In this case, all matching objects are rendered. "
```

Comment:
While you're at it, I think lines 61 and 62 should be one sentence, separated by a comma. The "In this case" is superfluous, IMHO. Same for lines 45, 46.

Reply:
Will do - question in between: up to now it seemed to automagically rebuild the `.ts` files, and spontaenously stopped now (likely because I'm not making changes that trigger the rebuild), I've looked at the build documentations and a bit at the cmake files etc. but I can't really figure out what's the one command I'm supposed to run to get all `.ts` files updated - do you have that in your head by any chance?

Reply:
Question resolved meanwhile, I extracted the `lupdate ...` command from somewhere and that one worked.

Reply:
Yeah, the translation generation is quite complex.
The problem is a circular dependency.
If you want to ensure that your translations are re-generated, you may want to remove all `*.qm` files and all compiled resource files (`qrc_resources*.cpp{,.o}`).
That of course does not save you from using `lupdate` to edit the translations manually.